pandas/io/pytables.py

```diff
@@ -1106,6 +1106,46 @@ def groups(self):
                  g._v_name != u('table'))))
         ]
 
+    def walk(self, where="/"):
+        """ Walk the pytables group hierarchy yielding the group name and
+        pandas object names for each group.  Any non-pandas PyTables objects
+        that are not a group will be ignored.
+
+        The where group itself is listed first (preorder), then each of its
+        child groups (following an alphanumerical order) is also traversed,
+        following the same procedure.  If where is not supplied, the root
+        group is used.
+
+        The where argument can be a path string
+        or a Group instance (see :ref:`GroupClassDescr`).
```

Comment:
what does this refer to? best to directly link to the pytables docs

Reply:
Actually it doesn't make sense for `where` to be anything else than `str`, removing this mention